docs/blog/to-be-collaborator.en-US.md

```diff
@@ -0,0 +1,153 @@
+git remote add upstream https://github.com/ant-design/ant-design
+```
+
+- Pull the latest remote branch version
+
+```markdown
+git pull upstream master
+```
+
+Now we are on the master branch from the fork, and this master is reserved for remote code tracking upstream
+
+### 3. Create a new fix branch on github
+
+![image-20221211130607684](https://user-images.githubusercontent.com/10607168/208016775-623abfe7-fa7f-438d-abc3-be445e52d8c5.png)
+
+### 4. Pull the new branch locally
+
+```
+git pull
+git checkout fix-branch
+```
+
+Now we can change the code on the branch
+
+## Find the issue on the [Ant Design issue](https://github.com/ant-design/ant-design/issues) page and analyze the selection
+
+To help you start your first attempt, we use [good first issues](https://github.com/ant-design/ant-design/issues?q=is%3Aissue+is%3Aopen+label%3A"good+first+issue") marks bugs and small features that are relatively easy to fix, and these issues are good as your first try. [help wanted](https://github.com/ant-design/ant-design/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) After all, it is easy to take over as a developer some problems.
+
+![image-20221216111126983](https://user-images.githubusercontent.com/10607168/208016864-fd72d378-a5db-4c20-9a34-b136d5e7c446.png)
+
+## Development Process
+
+1. `npm start` A website running Ant Design locally
+2. The specific development process, XXXX, not specifically written
+
+## Run test cases and specification checks
+
+1. Make sure all tests pass `npm run test`. Tips: You can use `npm test -- --watch TestName` to run specified tests during development.
+2. Run `npm test -- -u` to update [jest snapshot](https://facebook.github.io/jest/docs/en/snapshot-testing.html#snapshot-testing-with-jest) and put These updates are also committed (if any).
+3. Make sure your code passes the lint check `npm run lint`.
+
+## Merge changes
+
+- A common problem is that the remote upstream (ant-design/master) has a new update, which will lead to conflicts when we submit the Pull Request, so we can put the commits of other remote developers and Our commit is merged.
+
+- Switch to the `master` branch with the following code:
+
+```markdown
+git checkout master
+```
+
+- Use the following code to pull the latest code from the remote:
+
+```markdown
+git pull upstream master
+```
+
+- switch back to fix-branch
+
+```markdown
+git checkout fix-branch
+```
+
+- Merge the `commit` of `master` into `fix-branch`, and resolve conflicts if there are conflicts:
+
+```markdown
+git rebase master
+```
+
+- Submit the updated code to your own `fix-branch`:
+
+```markdown
+git push origin fix-branch
+```
+
+## Submit a Pull Request
+
+You can switch to the branches page on your github code warehouse page, click the fix-branch branch, click the `New pull request` button, add relevant comments and submit. Or switch to the code warehouse of the fix-branch branch, click the `Compare & pull request` button, add relevant comments and submit.
```

Comment:
I think by "warehouse" you mean "repository" no ?

Reply:
done